Corona-Warn-App/src/main/res/navigation/trace_location_attendee_nav_graph.xml

```diff
@@ -11,7 +11,7 @@
         tools:layout="@layout/fragment_confirm_check_in">
         <argument
             android:name="traceLocation"
-            app:argType="de.rki.coronawarnapp.ui.eventregistration.attendee.checkins.VerifiedTraceLocation" />
+            app:argType="de.rki.coronawarnapp.eventregistration.checkins.qrcode.VerifiedTraceLocation" />
```

Comment:
Thanks for moving it

Reply:
Why it is not in `ui` package though? it is parcelable entity for UI navigation purpose is not it?

Reply:
Kinda subjektive, but I'd have put it as a "core class" and not necessarily specific to UI, parcelable can also be passed to services. The whole package setup is currently a bit chaotic we will have to refactor some things once there are less moving parts and fewer chances for conflicts.

Based on @LukasLechnerDev  comments I've removed the whole class though.